src/reactor/mod.rs

```diff
@@ -548,15 +566,15 @@ impl Remote {
     ///
     /// Note that while the closure, `F`, requires the `Send` bound as it might
     /// cross threads, the future `R` does not.
-    pub fn spawn<F, R>(&self, f: F)
+    pub fn spawn<F, R>(&self, f: F) -> io::Result<()>
```

Comment:
Can you add documentation here for why `io::Result<()>` is returned? (e.g. what an error means)